spec/features/idv/analytics_spec.rb

```diff
@@ -28,6 +28,17 @@
       timed_out: false,
       transaction_id: 'ddp-mock-transaction-id-123' }
   end
+  let(:base_proofing_components) do
+    # rubocop:disable Layout/LineLength
+    { document_check: 'mock', document_type: 'state_id', source_check: 'aamva', resolution_check: 'lexis_nexis', threatmetrix: threatmetrix, threatmetrix_review_status: 'pass' }
+    # rubocop:enable Layout/LineLength
```

Comment:
I'd just as soon _not_ disable the LineLength linter, but that's the style in this file, Kyle.

Reply:
seems like this part could benefit from being broken out onto separate lines?

Reply:
Woot, that was actually my preference but I had my silly rhyme about that being the norm in this file. There's no reason to keep it that way, so update incoming.